### ISSUE TITLE
Incorrect scroll speed in some windows

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#2574]
see #2264

**Title screen, main menu screen, party/actor battle menus**
- Scrolling stops at end of menu. In RPG_RT, holding down arrow scrolls endlessly.
- Scrolling might be _slightly_ too slow.

**Equip screen**
![](https://i.imgur.com/kWS4l0y.png)
- Same as above
- Scrolling stops at end of equip slots. In RPG_RT, holding down arrow scrolls endlessly.

**Shop screen**
![](https://i.imgur.com/MgLoO2y.png)
- No smooth scrolling (not in other menus?)

**Comment by rueter37** [comment]
Hasn't the endless scroll issue already been fixed? Or do I miss something?

**Comment by Ghabry** [comment]
I do not understand "task lists". I added the issue but the task was still not flagged as "done" after merge -_-

@rueter37  is anything else of this already fixde?

**Comment by rueter37** [comment]
@Ghabry Yes, the checkboxes 1 and 3 are already fixed by #2361. Only "smooth scrolling" (checkbox 4) is remaining.
